data/frm/Crecy_1997_xPonthus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+digraph {
+
+x [color="grey"];
```

Comment:
Are they all direct descendant of x ?

Looking at the stemma (but I do not have the text), I would be tempted to encode them as 
```gv
x->1;
1->2;
2->P1;
2->P2;
1->P3;
1->P4;
```

Reply:
The manuscripts P1, P2, P3, P4 are all at the same level and form a group (a), which is not shown in the stemma.

Should we create a node 1 (x->1), to represent that?

Reply:
Yes, I think so. I'm adding it.